extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/templates/deployment.yaml

```diff
@@ -116,7 +119,7 @@ spec:
         {{- if not (eq .Values.sparkConf "") }}
         - name: spark-config
           configMap:
-            name: {{ .Release.Name }}-shs-spark-config
+            name: spark-history-service-spark-config
```

Comment:
Q: What's the thought process behind leaving this hard-coded? I'd think we'd still use app/chart name.

Reply:
Configmap is used in deployment yaml and doent strictly need to have the release/app name in it. Seen it both  hardcoded and customizable in other modules. Happy to add the app name if we think that would make it more clear

Reply:
I don't think it'd be advisable in the vast majority of situations, but if you're in a spot where you _need_ to deploy two instances of this chart to the same namespace, you'd need to be able to differentiate.

I don't think it's the biggest deal since the vast majority of the time projects should be segregating app deployments by k8s namespace. But there's always a chance there's some edge case we hadn't considered.

Reply:
Yeah I think thats good justification. Ill update